README.md

```diff
@@ -1,6 +1,6 @@
 # jOCD
 
-jOCD is a port to JAVA/Android from pyOCD (https://github.com/mbedmicro/pyOCD) project. This is a JAVA library for programming and debugging ARM Cortex-M microcontrollers using CMSIS-DAP. Currently, only Android is supported. Windows, Linux and OSX might become supported.
+jOCD is a port to JAVA/Android from pyOCD (https://github.com/mbedmicro/pyOCD) project. This is a JAVA library for programming Microbit board using CMSIS-DAP. Currently, only Android is supported. Windows, Linux and OSX might become supported.
```

Comment:
Some suggestions:
- Java instead of JAVA.
- I think we could just say that "jOCD is Java port of the pyOCD (https://github.com/mbedmicro/pyOCD) project".
- Missing ":" in the name of micro:bit board. Also, the "m" seems to be lowercase.

Reply:
Ok, I've merged with these changes.